x-pack/plugins/alerting/server/alert/alert.ts

```diff
@@ -292,8 +291,8 @@ export class Alert<
     // Related issue: https://github.com/elastic/kibana/issues/144862
 
     return !summarizedAlerts.all.data.some(
-      (alert) =>
-        get(alert, ALERT_UUID) === this.getId() || get(alert, ALERT_UUID) === this.getUuid()
+      (alert: AlertHit) =>
+        alert?.kibana?.alert?.uuid === this.getId() || alert?.kibana?.alert?.uuid === this.getUuid()
```

Comment:
This should always be populated since it's a required field but I always get nervous accessing deeply nested objects hence the `?`s. I can remove if we feel it's unnecessary